v4/source/introduction.yaml

```diff
@@ -263,7 +260,7 @@ tags:
       - channel_deleted
         ```
         {
-          'event': 'WebsocketEvents.CHANNEL_DELETED',
+          'event': '<event_string>',
```

Comment:
`WebsocketEvents.CHANNEL_DELETED` actually stands for a real string: `channel_deleted`

You can see what the other ones are here https://github.com/mattermost/mattermost-server/blob/master/model/websocket_message.go